modules/table/src/testFixtures/java/org/apache/ignite/internal/table/TxAbstractTest.java

```diff
@@ -276,17 +291,21 @@ protected TxManager clientTxManager() {
 
     /** {@inheritDoc} */
     protected TxManager txManager(TableViewInternal t) {
+        TxManager manager = txTestCluster.txManagers().get(primaryNode(t));
```

Comment:
Please add assertNotNull for primaryNode(t) result. Formally it's valid to have null as getPrimaryReplica result, but likely it's not the case within given tests.